language/lexer/lexer.go

```diff
@@ -237,6 +240,7 @@ func readString(s *source.Source, start int) (Token, error) {
 					valueBuffer.WriteRune('"')
 					break
 				case '/':
+					valueBuffer.WriteRune('\\')
```

Comment:
Hmm, that double escaping is tricky to see and get right.
I think you might use raw string literals in backticks to make it a little easier to understand :)
Like

    Body: `"slashes \\ \/"`,

However I just ask myself why and if `/` is escaped at all.